gsa/src/web/pages/results/detailspage.js

```diff
@@ -238,15 +238,18 @@ const Details = ({
           title={_('Tags')}
         >
           <Divider>
-            {userTags.map(tag => (
-              <DetailsLink
-                key={tag.id}
-                id={tag.id}
-                type="tag"
-              >
-                {tag.name + '=' + tag.value}
-              </DetailsLink>
-            ))}
+            {userTags.map(tag => {
+              const valueString = tag.value === '' ? '' : '=' + tag.value;
```

Comment:
Could you use  `isEmpty(tag.value)` from gmp/utils/string instead?

Reply:
Maybe it would even better to parse the tag value as undefined if it is an empty string...